.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Explicitly declare text files you want to always be normalized and converted to native line endings on checkout.
+*.ts text eol=lf
+*.tsx text eol=lf
```

Comment:
nit: missing newline at end of file

---

can this just be `*` for all files ... ?

```suggestion
* text eol=lf
```

Reply:
I tested it out by removing the `*.ts` and `*.tsx` and replacing them with `* text`. And it modified the `.png` files as screenshot below. Maybe specifying the files is necessary here?

![text gitattributes](https://github.com/open-sauced/ai/assets/45172775/7eea7aaf-f664-40bb-af3f-ad032eecb06e)

Reply:
@adiati98, what I meant was, keep the existing ones and add other file types we want to handle, e.g.

```diff
*.ts text eol=lf
*.tsx text eol=lf
+ *.json text eol=lf
+ *.md text eol=lf
```

Reply:
Sorry for misunderstanding it, @nickytonline!
I'll do the fix. 😊

Reply:
@jpmcb I made the fix following suggestion from @nickytonline.

You mentioned:

> nit: missing newline at end of file

I added a new line at the end of file, but it's somehow not reflected in the "File changed" tab.

Below are screenshots from VS Code and edit file on GitHub that shows that this file contains 7 lines:

![gitattributes vscode](https://github.com/open-sauced/ai/assets/45172775/7ec65c99-df27-40f6-8e72-fee441b3d724)

![gitattributes gh](https://github.com/open-sauced/ai/assets/45172775/814c5fab-e6d9-4195-a562-9ec3264b9a9a)